README.md

```diff
@@ -66,6 +101,13 @@ make up
 You can now connect to your local Airflow web console at
 `http://localhost:8000/`.
 
+All DAGs are paused by default for local instances and our staging instance of Airflow.
+In order to submit a DAG via the UI, you'll need to toggle the DAG from "Off" to "On",
+but check to see what DAG runs are generated (Browse > DAG Runs), since it may start
+generating backfill runs based on the DAG's configured start date
+(set `schedule_interval=None` in your DAG definition to prevent these scheduled runs). 
+You'll likely want to toggle the DAG back to "Off" as soon as your desired task starts running.
```

Comment:
There should be a larger disclaimer here, turning on the DAG and accidentally backfilling is something we want people to avoid doing.